src/components/BlockingViews/BlockingView.js

```diff
@@ -37,6 +48,15 @@ const BlockingView = props => (
         />
         <Text style={[styles.notFoundTextHeader]}>{props.title}</Text>
         <Text style={[styles.textAlignCenter]}>{props.subtitle}</Text>
+        {!!props.shouldShowBackHomeLink
```

Comment:
why !! even after we gave a default prop?

Reply:
Yup default prop is good, but the main concern when using some text with condition it safe to use !! - to avoid any potential crash, if some prop will be mixed/not used correctly from parent.

Reply:
I dont think we use this in our codebase anywhere, could we explore some other way to achieve this jsut so we dont introduce this pattern here? Thanks!

Reply:
pushed changes thanks!